test/fixtures/served/ampdoc-with-messaging.html

```diff
@@ -22,6 +22,7 @@
   <meta name="viewport" content="width=device-width,minimum-scale=1">
   <script async src="../../../dist/amp.js"></script>
   <script async src="../../../dist/v0/amp-viewer-integration-0.1.js"></script>
+  <script async src="../../../dist/v0/amp-viewer-integration-0.1.max.js"></script>
```

Comment:
I don't think this would do what you want it to do. Basically, with this, you don't know which version is picked up in either case - that creates a flaky scenario. Also, you have other max scripts here - e.g. `amp.js`. I'd say we either need to force everything to be `max` here; or create two files.